eslint.config.mjs

```diff
@@ -0,0 +1,145 @@
+import eslint from "@eslint/js";
+import tseslint from "@typescript-eslint/eslint-plugin";
+import tsParser from "@typescript-eslint/parser";
+import prettierConfig from "eslint-config-prettier";
+import i18nextPlugin from "eslint-plugin-i18next";
+import mdxPlugin from "eslint-plugin-mdx";
```

Comment:
```suggestion
```
we don't use mdx in care_fe, so this can be dropped, remove the dependency from package.json as well